packages/design-system/src/components/contextMenu/menu.js

```diff
@@ -37,13 +37,15 @@ import {
 import { useKeyDownEffect } from '../keyboard';
 import { useContextMenu } from './contextMenuProvider';
 
+export const CONTEXT_MENU_WIDTH = 218;
+
 const MenuWrapper = styled.div(
   ({ theme }) => css`
     padding: ${({ $isIconMenu }) => ($isIconMenu ? '4px 3px' : '8px 0')};
     background-color: ${theme.colors.bg.primary};
     border-radius: ${theme.borders.radius.small};
     border: 1px solid ${theme.colors.border.disable};
-    width: ${({ $isIconMenu }) => ($isIconMenu ? 40 : 210)}px;
+    width: ${({ $isIconMenu }) => ($isIconMenu ? 40 : CONTEXT_MENU_WIDTH)}px;
```

Comment:
This change applies to all use cases now though, is that intended?

Reply:
I don't know where that 210 comes from ([this Figma file](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=5034%3A442122) suggests variable width), but change to 218 does only good things (I think):
210:
![image](https://user-images.githubusercontent.com/21036224/150527170-18a31569-6612-4aa5-9204-7816fa78c332.png)

218:
![image](https://user-images.githubusercontent.com/21036224/150527258-32d00d64-6734-4824-91c1-fff82a698d16.png)

extra points for making @barklund very happy 😄

Reply:
🎯

Reply:
Done: https://github.com/GoogleForCreators/web-stories-wp/issues/10365

Reply:
210 just came from figma. If it looks good larger then that's fine too 👍